src/main/scala/ch/epfl/bluebrain/nexus/kg/search/QueryBuilder.scala

```diff
@@ -7,12 +7,12 @@ import io.circe.Json
 object QueryBuilder {
 
   private def baseQuery(filterTerms: List[Json]): Json =
-    Json.obj(
-      "query" -> Json.obj(
-        "bool" -> Json.obj(
-          "filter" -> Json.arr(filterTerms: _*)
-        )
-      ))
+    Json.obj("sort" -> Json.arr(Json.fromString("@id")),
```

Comment:
Do we want to sort by id? is a it useless since the id's are mostly randomly generated

Reply:
We need to have some sorting, so that the results are consistent. We could also sort  by time created/updated by default, but I'm not sure which one is better.

Reply:
maybe time created by default? Seems like a reasonable one?

Reply:
👍